infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -156,20 +156,17 @@
                     + "                                                                          "
                     + "              [default: TLSv1.1,TLSv1.2].\n"
                     + " --base-server-threads=<base_server_threads>                              "
-                    + "              Number of threads dedicated for the base server.\n          "
+                    + "              Number of threads dedicated for the base server [default: 0].\n"
                     + " --log-size-quota-percentage=<max_log_size_percentage>                    "
                     + "              The max size as percentage of underlying file-store size.\n "
                     + "              If this limit is exceeded "
                     + "              write requests will be rejected [default: 100.0].\n         "
                     + "                                                                          "
-                    + " --layout-server-threads=<layout_server_threads>                          "
-                    + "              Number of threads dedicated for the layout server.\n        "
-                    + "                                                                          "
```

Comment:
Just a question, is this because the LayoutServer executor is single-threaded?

Reply:
Yes.